spec/models/portfolio_spec.rb

```diff
@@ -18,6 +18,35 @@
     end
   end
 
+  context "model fails validation on unique name by same tenant" do
```

Comment:
Great specs, but I think we can rework the `context` language and simplify the `it` method names. [This](http://www.betterspecs.org/) is a pretty good guide and/or reference.

Reply:
Got it reworded.